### PR TITLE
Homework for JS1 week3

### PR DESCRIPTION
**Pull request by veronicaburduzhan**

### CODE REVIEW

File: javascript/javascript1/week3/homework/main-5.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+console.log(activities);
+
+//SHOWING STATUS OF SCREEN TIME ON A CERTAIN DATE
+function showStatus(date) {
+  let totalDurationStatus = 0;
+  let numberActivities = 0;
+  const limit = 100;
+  for (i = 0; i < activities.length; i++) {
+    if (activities[i].date == date) {
```
Comment (guillermoFragachan) [inline javascript/javascript1/week3/homework/main-5.js:31]: Try to use the triple equal("===") as a comparison operator, if the types a different is always better to parse number to string or vice-versa than using the double equal ("=="), it won't have an effect here but it is a good practice to have 😊

**Review by guillermoFragachan — APPROVED** [review]
Everything good

File: javascript/javascript1/week3/homework/main-1.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "Ahmad",
+  "Yana",
+  "Kristina",
+  "Rasmus",
+  "Samuel",
+  "Katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+const removedItem = names.splice(names.indexOf("Ahmad"), 1)
```
Comment (Falafelqueen) [inline javascript/javascript1/week3/homework/main-1.js:12]: Could you think of a way to make your code more dynamic ? Right now it removes only "Ahmad" and it won't work when the nameToRemove changes :-)

File: javascript/javascript1/week3/homework/main-3.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+});
+seriesDurations.push({
+  title: "Behind Her Eyes",
+  days: 0,
+  hours: 5,
+  minutes: 0,
+});
+seriesDurations.splice(0, 3);
+console.log(seriesDurations);
+
```
Comment (Falafelqueen) [inline javascript/javascript1/week3/homework/main-3.js:42]: Cool ! You could have also manually replace the example series with your favorite ones, but this is works great 🏋️‍♀️

File: javascript/javascript1/week3/homework/main-3.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      console.log(
+        `${series.title} took ${spentTimeForSeries.toFixed(3)}% of my life`
+      );
+      totalSpentTimeForSeries += spentTimeForSeries;
+    });
+    console.log(`In total that is ${totalSpentTimeForSeries.toFixed(3)}% of my life`)
+}
+
+logOutSeriesText(seriesDurations, 80);
```
Comment (Falafelqueen) [inline javascript/javascript1/week3/homework/main-3.js:59]: Great ! However, in the homework assignment it says also the time spent for each series, could you implement that as well ? :)

**Review by Falafelqueen — APPROVED** [review]

File: javascript/javascript1/week3/homework/main-1.js (hunk 2 above)
Comment (veronicaburduzhan) [reply javascript/javascript1/week3/homework/main-1.js:12]: Do you mean replace const with let?

File: javascript/javascript1/week3/homework/main-3.js (hunk 3 above)
Comment (veronicaburduzhan) [reply javascript/javascript1/week3/homework/main-3.js:42]: Thanks:)

File: javascript/javascript1/week3/homework/main-5.js (hunk 1 above)
Comment (veronicaburduzhan) [reply javascript/javascript1/week3/homework/main-5.js:31]: Thanks for the great advice:) P.S. I usually use triple equal but why I did double in this case I cannot explain. I finished this exercise more than 2 weeks ago.

File: javascript/javascript1/week3/homework/main-3.js (hunk 4 above)
Comment (veronicaburduzhan) [reply javascript/javascript1/week3/homework/main-3.js:59]: But I did it. You can see it in line 52. In console it looks like this:
_Dark took 0.003% of my life
Castle took 0.019% of my life
Behind Her Eyes took 0.001% of my life
In total that is 0.022% of my life_

**Pull request merged**